src/coreclr/jit/unwind.cpp

```diff
@@ -187,7 +187,12 @@ void Compiler::unwindPushPopMaskCFI(regMaskTP regMask, bool isFloat)
     regMaskTP regBit = isFloat ? genRegMask(REG_FP_FIRST) : 1;
 
     for (regNumber regNum = isFloat ? REG_FP_FIRST : REG_FIRST; regNum < REG_COUNT;
-         regNum           = REG_NEXT(regNum), regBit <<= 1)
+#if TARGET_ARM
```

Comment:
For better readability, I would rewrite this to.  I would add a short comment for why we need to do special case for ARM.

```c#
regNumber regNum = isFloat ? REG_FP_FIRST : REG_FIRST;
for (; regNum < REG_COUNT;)
{
#if TARGET_ARM
    regNum = isFloat ? REG_NEXT(REG_NEXT(regNum)) : REG_NEXT(regNum);
    regBit <<= isFloat ? 2 : 1;
#else
    regNum = REG_NEXT(regNum);
    regBit <<= 1;
#endregion
...
...
}

```

Reply:
Done!